### PR TITLE
Add example jupyter notebooks

### PR DESCRIPTION
**Pull request by 1duo**

### CODE REVIEW

**Comment by 1duo** [comment]
This also resolves #355

File: examples/readme.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+## Convert TensorFlow model to Core ML model format
```
Comment (aseemw) [inline examples/readme.md]: now that you have moved this description in its own readme which is good, can you delete the descriptions from the top level readme?

Comment (1duo) [reply examples/readme.md]: Sure!

Comment (1duo) [reply examples/readme.md]: Done.

**Pull request merged**